### ISSUE TITLE
GAWB-3316: update free credit spreadsheet to have "50" in extra credits

### ISSUE THREAD

**Issue by irosenbe** [DataBiosphere/firecloud-app#28]
automatically add "50" to the "extra credits" column (column Q) of the Onix Free Credits spreadsheet for every new user that is matched with a billing project for the free credit program. this will enable us to expand the free credit program to $300 per user.

production spreadsheet:
https://docs.google.com/spreadsheets/d/1Uj_AwH9A9I6rw7NpVsT0FegGSF5Q9gS268YNe7TJQvg/edit?ts=5a551f1a#gid=0

Implementation notes:
The APIs you care about (dev env) are:

https://firecloud-orchestration.dsde-dev.broadinstitute.org/#!/Trial_(FireCloud_Credits)/createReportTrialProjects - create a new spreadsheet to play around with
https://firecloud-orchestration.dsde-dev.broadinstitute.org/#!/Trial_(FireCloud_Credits)/updateReportTrialProjects - update an existing spreadsheet.

We (Ilyana) calls the update endpoint daily to keep the production spreadsheet up-to-date (see ticket related to automate updates to onix billing report for avoiding this manual step). The "extra credits" column in the production spreadsheet is a column we manually added and is not touched by the update API - that's why you won't see it if you create a new spreadsheet. Campaign managers (Ilyana, Tiffany) also manually edit this column when they want to grant users additional credit dollars.

Since the extra credits column is manually maintained and not touched by the update API, I suggest the implementation does NOT involve any Scala changes and can be done entirely in the spreadsheet itself. Make sure any existing users who have a blank in that column remain at zero extra credits; make sure that any new users added to the spreadsheet get 50.

Link to GAWB ticket: https://broadinstitute.atlassian.net/browse/GAWB-3316

**Comment by irosenbe** [comment]
4/11: On hold pending confirmation from Onix that this is the correct route in which to proceed

**Comment by davidangb** [comment]
Because we only have one tracking spreadsheet that we care about, this is already in production, but I'm leaving it here until someone reviews. 

How I handled this:
* I did not touch any existing users who had a value (e.g. 250) in the Extra Credits column
* existing users who had a blank in the Extra Credits column now have a 0
* any rows that did not have a user assigned (i.e. available projects in the pool) now have a 50 in the Extra Credits column
* I added 500+ rows with a 50 in the Extra Credits column to handle any newly-created projects
